src/librustc/hir/lowering.rs

```diff
@@ -1664,7 +1664,7 @@ impl<'a> LoweringContext<'a> {
                         if pos == ImplTraitPosition::Binding &&
                             nightly_options::is_nightly_build() {
                             help!(err,
-                                  "add #![feature(impl_trait_in_bindings)] to the crate attributes \
+                                  "add `#![feature(impl_trait_in_bindings)]` to the crate attributes \
```

Comment:
This line is too long.

Reply:
Yeah. The cap is 100 chars. You can run `./x.py test src/tools/tidy` locally to check this.

Reply:
My apologizes. That's noted! I will check this